build.rs

```diff
@@ -0,0 +1,85 @@
+use std::env;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+use maplit::hashmap;
+use memofs::VfsSnapshot;
+
+fn snapshot_from_fs_path(path: &PathBuf) -> Result<VfsSnapshot, PathBuf> {
```

Comment:
`PathBuf` is a strange error type! We should use `io::Error` (by returning `io::Result<VfsSnapshot>`) I imagine.